api.planx.uk/helpers.ts

```diff
@@ -83,53 +87,42 @@ const createAssociatedOperation = async (flowId: Flow["id"]) => {
     },
   );
 
-  return data?.insert_operations_one;
+  return data?.operation;
 };
 
+interface PublishedFlows {
+  flow: {
+    publishedFlows: {
+      // TODO: use FlowGraph from planx-core here
```

Comment:
This `TODO` is intentionally left here - making this change has pretty messy cascading effects that would add a lot of noise to this PR.

I'll make a GitHub issue describing the required change here.

Reply:
Issue logged here - https://github.com/theopensystemslab/planx-new/issues/2405